### PR TITLE
Limit UKCP Tokens to 1

### LINKED ISSUE #1562: As a user, I only need one UKCP API key
Look at allowing users just 1 key, and prompt to redownload / delete key when requesting a new one.

Linked UKCP API issue: https://github.com/VATSIM-UK/uk-controller-api/issues/41

### PR DESCRIPTION
**Pull request by atoff**
Refactors functionality to allow users only one API token.

As a note, instead of enforcing a hard limit, I have maintained the HasMany relationship style, so that people with existing multiple tokens aren't suddenly effected (and in case we ever want to do some with multiple tokens again). Over time, people with multiple tokens should be phased out naturally.

image

Closes #1562

### CODE REVIEW

File: app/Http/Controllers/UKCP/Token.php [hunk 1]
```diff
@@ -35,47 +34,31 @@ public function create()
             return Redirect::route('mship.manage.dashboard')
                 ->withError('An unknown error occured, please contact Web Services.');
         }
-
-        $latestId = $this->ukcp->getValidTokensFor(auth()->user())->first()->id;
-        $tokenPath = 'ukcp/tokens/' . auth()->user()->id . '/' . $latestId . '.json';
-        Storage::disk('local')->put($tokenPath, $newToken);
-
-        return $this->viewMake('ukcp.token.create')->with('newToken', $latestId);
+        return redirect()->route('ukcp.guide')->withSuccess('Tokens Updated!');
```
Comment (AxonC) [inline app/Http/Controllers/UKCP/Token.php]: Do you really want to go back to the guide?

Comment (atoff) [reply app/Http/Controllers/UKCP/Token.php]: I think this is best. Provides a clear way to download the new key (the big download button), and for new people they don't need to click the link to view what they need to do (= hopefully less questions!)

Comment (atoff) [reply app/Http/Controllers/UKCP/Token.php]: And it already did go back to the guide (ukcp.token.create is the view that is now renamed ukcp.token.guide, and the ukcp.guide route makes that view)

File: app/Libraries/UKCP.php [hunk 2]
```diff
@@ -3,31 +3,44 @@
...
 use App\Models\Mship\Account;
+use Bugsnag\BugsnagLaravel\Facades\Bugsnag;
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\ClientException;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Psr\Http\Message\ResponseInterface;
 
 class UKCP
```
Comment (AxonC) [inline app/Libraries/UKCP.php]: Might be worth creating a singleton in the AppServiceProvider to auto-resolve the dependency injection

Comment (atoff) [reply app/Libraries/UKCP.php]: Did this originally, it doesn't make a difference. The dependency is automatically resolved anyway

Comment (AxonC) [reply app/Libraries/UKCP.php]: that would be the major motivation behind it

**Pull request merged**